_search-plugins/knn/approximate-knn.md

```diff
@@ -25,8 +25,7 @@ and then use this search method. All filters are applied on the results produced
 
 Each of the three engines used for approximate k-NN search has its own attributes that make one more sensible to use than the others in a given situation. You can follow the general information below to help determine which engine will best meet your requirements.
 
-* The faiss engine performs exceptionally well (on orders of magnitude) with hardware that includes a GPU. When cost is not the first concern, this is the recommended engine.
```

Comment:
"In general, nmslib outperforms both faiss and Lucene on search. However, to optimize for indexing throughput, faiss is a good option."